web/src/app/projects/list/list-my-projects.component.html

```diff
@@ -0,0 +1 @@
+<dashboard-projects-private-public [title]="title" [projects]="projects" showTitle="true"></dashboard-projects-private-public>
```

Comment:
I think `showTitle` is always passed as string here. I believe it should be boolean

Reply:
good spot, I will update